### PR TITLE
Bug: work on PR reviews

### PR DESCRIPTION
**Pull request by MikeManzi**
### What does this PR do?
- Addresses the reviews from the staging PR

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,61 +1,110 @@
...
+              "yamo",
+            ];
+
+            listItems.map((item) => {
+              const li = document.createElement("li");
+              li.textContent = item;
+              li.classList.add("cursor-pointer")
+              ul.appendChild(li);
+            });
+          }
```
Comment (VictorMugisha) [inline index.html:107]: I like the way you did this!

**Review by VictorMugisha — APPROVED** [review]
It is now looking good and clean!!

**Review by kelliaUmuhire — APPROVED** [review]
Looks amazing!

**Review by tuyishimehono — APPROVED** [review]
Looks good! The code is well formatted now.

**Pull request merged**